src/common/KokkosKernels_IOUtils.hpp

```diff
@@ -169,7 +169,8 @@ void kk_diagonally_dominant_sparseMatrix_generate(
     OrdinalType bandwidth,
     ScalarType* &values,
     SizeType* &rowPtr,
-    OrdinalType* &colInd)
+    OrdinalType* &colInd,
+    ScalarType diagDominance = 10 * Kokkos::ArithTraits<ScalarType>::one())
```

Comment:
@seheracer Brian's change here may be helpful for your detective work with float type in spgemm_jacobi (I think you had observations about the scalar multiplier's value on diagonally dominant matrices having impact on convergence)

Reply:
Thanks, @ndellingwood for letting me know. This is a useful feature @brian-kelley, thanks!